composer.json

```diff
@@ -39,6 +39,7 @@
         "sonata-project/cache-bundle": "^2.1",
         "sonata-project/doctrine-phpcr-admin-bundle": "1.2.*",
         "sonata-project/admin-bundle": "^2.3.3",
+        "sonata-project/translation-bundle": "1.0.0",
```

Comment:
Like all Sonata packages, `1.0.1` contained a major BC break and is not usable with SonataBlockBundle 2.2.